OpenRA.Game/FieldLoader.cs

```diff
@@ -524,11 +507,9 @@ public static object GetValue(string fieldName, Type fieldType, MiniYaml yaml, M
 				if (value != null)
 				{
 					var parts = value.Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries);
-					float x = 0;
-					float y = 0;
 					float z = 0;
```

Comment:
Imo it would make more sense now to declare this after the comment about the "z component" below.